app/server/src/main/scala/org/bitcoins/server/BitcoindRpcBackendUtil.scala

```diff
@@ -81,7 +86,7 @@ object BitcoindRpcBackendUtil extends Logging {
 
       val hasFiltersF = bitcoind
         .getFilter(wallet.walletConfig.chain.genesisHashBE)
-        .map(_ => true)
+        .map(_ => false)
```

Comment:
This skips filter sync when we have a compatible bitcoind node. This should slow down the wallet IBD. We need this so we emit a `blockprocessed` message for every block header, rather than just ones that are compact filter matches.

Reply:
this will greatly slow down the sync time

Reply:
IIRC changing from blocks to filters improved sync time by like 5-10x

Reply:
I reverted this, but the tradeoff here is that we do not propagate `blockprocessed` events when doing IBD with the bitcoind backend. When our wallet is in steady state, it will execute callbacks for `blockprocessed`.